Update/omot_003.txt

```diff
@@ -930,14 +930,14 @@ void main()
 			NULL, "\"Anyway, after that baseball game a little while back...", Line_WaitForInput);
 	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600166", 256, TRUE);
 	OutputLine(NULL, "圭ちゃんがきっかけで話をしたんです」",
-			NULL, " Satoko was having a chat with Kei-chan.\"", Line_Normal);
+			NULL, " I had a chance to chat with Satoko thanks to Kei-chan.\"", Line_Normal);
 	ClearMessage();
 
 //詩音rvS14/06/210600167.「で、ちょっと意地悪な気持ちで質問してみました。kvS14/06/210600168.『悟史くんが帰ってきたら、真っ先に何をしてあげます？』って。kvS14/06/210600169.…その時にあの子、なんて答えたと思います？」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600167", 256, TRUE);
 	OutputLine(NULL, "「で、ちょっと意地悪な気持ちで質問してみました。",
-			NULL, "\"And, he was lightly teasing her while asking a question.", Line_WaitForInput);
+			NULL, "\"So, I was lightly teasing her while asking a question.", Line_WaitForInput);
```

Comment:
Mistake. "So, I tried lightly teasing her while asking a question." is accurate.

Reply:
```suggestion
			NULL, "\"So, I tried lightly teasing her while asking a question.", Line_WaitForInput);
```